prototype_source/openvino_quantizer.rst

```diff
@@ -0,0 +1,245 @@
+PyTorch 2 Export Quantization for OpenVINO torch.compile backend.
+===========================================================================
+
+**Authors**: `Daniil Lyakhov <https://github.com/daniil-lyakhov>`_, `Alexander Suslov <https://github.com/alexsu52>`_, `Aamir Nazir <https://github.com/anzr299>`_
+
+Prerequisites
+--------------
+- `PyTorch 2 Export Post Training Quantization <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq.html>`_
+- `How to Write a Quantizer for PyTorch 2 Export Quantization <https://pytorch.org/tutorials/prototype/pt2e_quantizer.html>`_
+
+Introduction
+--------------
+
+This tutorial demonstrates how to use `OpenVINOQuantizer` from `Neural Network Compression Framework (NNCF) <https://github.com/openvinotoolkit/nncf/tree/develop>`_ in PyTorch 2 Export Quantization flow to generate a quantized model customized for the `OpenVINO torch.compile backend <https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html>`_ and explains how to lower the quantized model into the `OpenVINO <https://docs.openvino.ai/2024/index.html>`_ representation.
```

Comment:
It would be more attractive if to give the user an idea why it may need to use OpenVINOQuantizer (e.g. it is more accurate, performant, etc.)

Reply:
Make sense! Description of advantages of OpenVINOQuantizer was added